src/main/java/ch/njol/skript/util/ColorRGB.java

```diff
@@ -32,7 +32,7 @@
 
 public class ColorRGB implements Color {
 
-	private static final Pattern RGB_PATTERN = Pattern.compile("(?>rgb|RGB) (\\d+), (\\d+), (\\d+)");
+	private static final Pattern RGB_PATTERN = Pattern.compile("(?>rgb|RGB) (\\d+), ?(\\d+), ?(\\d+)");
```

Comment:
i mean i guess but this seems like a regressive change to me. Yes it makes things slightly easier for users but I don't think we should be encouraged less space usage where we don't have to. opinions welcome